sri/templatetags/sri.py

```diff
@@ -8,28 +8,42 @@
 from django.utils.safestring import mark_safe
 
 from sri.algorithm import DEFAULT_ALGORITHM, Algorithm
+from sri.extra_attribute import ExtraAttribute
 from sri.integrity import calculate_integrity_of_static
 
 USE_SRI = getattr(settings, "USE_SRI", not settings.DEBUG)
 
 register = template.Library()
 
 
-def sri_js(attrs: dict, path: str, algorithm: Algorithm):
-    attrs.update({"type": "text/javascript", "src": static(path)})
+def sri_js(attrs: dict, path: str, algorithm: Algorithm, **kwargs):
+    if 'extra_attr' in kwargs:
```

Comment:
I wonder if instead of `extra_attr`, the template tags should just be called with whichever of these extra attributes they need? Might make the API cleaner

Reply:
The thing is that if it's the case, I think we will encounter difficulties due to the use of positional arguments. We would need to give the algorithm value _then_ specify an extra attribute (you cannot specify an extra attribute without specifying the algorithm first). And then we would need to write logic to allow only one extra attribute to be specified. Tell me if I am wrong, I'm kinda new to programming.

Reply:
I believe you can still do `{% sri_static "file.js" "sha256" "async" "defer" %}` for example?